src/desktop/lib/global_client_setup.coffee

```diff
@@ -27,7 +27,7 @@ module.exports = ->
   setupErrorReporting()
   setupJquery()
   setupReferrerTracking()
-  syncAuth()
+  # syncAuth()
```

Comment:
Did we undo this somehow? I though you've already made this fix before @mzikherman ?

Reply:
Maybe worth putting a comment to note why this `sync` is problematic.

Reply:
> Did we undo this somehow? I though you've already made this fix before @mzikherman ?

Yup, I linked and mentioned it in the PR description :)

Yes will add a comment!